e2e/cypress/integration/system_console/site_url_config_spec.js

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
```

Comment:
Should the stage be set? I thought they usually set it after the e2e test has been determined to not be flaky

Reply:
@saturninoabril Do you know the answer to this? Should I be setting this or should I leave it blank by default?

Reply:
Yeah, please leave it blank as default. Once merged, it will run together with other unstable tests for several days and will move to prod once we found it to be stable.

Reply:
Will do; thanks Saturn. Thanks for noticing this Farhan.